### ISSUE TITLE
UserWarning has reference to removed twisted.im

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5598]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5598|
|Type|defect|
|Created|2012-04-02 19:05:00Z|

There's a reference in t.words.im that references the old `twisted.im` package, instead of `twisted.words.im`.

```
$ python doc/words/examples/minchat.py 
/home/thijs/workspaces/opensource/software/twisted/svn/Twisted/trunk/twisted/words/im/__init__.py:8: UserWarning: twisted.im will be undergoing a rewrite at some point in the future.
```

Attachments:

* [warrningCorrection.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b56/b567e1aae66398cae61014d54a01335f6843ceb9/ddf1a12713086d54948527c1c1b0949bcae07277.diff) (707 bytes) - added by chhabraamit on 2012-04-05 23:03:08Z - 
* [5598.removal](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b56/b567e1aae66398cae61014d54a01335f6843ceb9/9fa88fc1ff9a20ab9c807aa249a20693eb10cb9c.removal) (93 bytes) - added by chhabraamit on 2012-04-05 23:05:43Z - 
* [newCorrectedPatch.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b56/b567e1aae66398cae61014d54a01335f6843ceb9/9c959513c3defe93500002fcd6b2c7f1f7d4536d.diff) (689 bytes) - added by chhabraamit on 2012-04-05 23:48:25Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5598 5598
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__easy easy
time__1333393500000000 1333393500000000
changetime__1333712494000000 1333712494000000
version__None None
owner__exarkun exarkun
cc__thijs cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

This warning should just be removed.  We are not going to rewrite twisted.im or twisted.words.im; we will maintain it like everything else.  If something specific will be deprecate it we can add a more specific warning at that time.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Thanks for working on this ticket, chhabraamit.

The change to `__init__.py` looks great.  I don't see any problems with it.  The news file doesn't look quite right though.  We don't really have a convention for removing deprecation warnings, since we don't do it very often.  However, I think that the right thing to do here is to include a bugfix fragment.  Nothing _important_ or _useful_ is being removed here.  Instead, a bug - a warning that should not be emitted - is being fixed.

I'll adjust the news fragment and apply the patch.  Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

One other point - news fragments go into the project or subproject where the change is being made.  This change is made to Twisted Words, so the news fragment belongs in twisted/words/topfiles/.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [34100]) Remove rewrite warning from `twisted.words.im` package.

Author: chhabraamit
Reviewer: exarkun
Fixes: [#5598](https://github.com/twisted/twisted/issues/5598)

Twisted Words is maintained like any other part of Twisted, with incremental improvements
and backwards compatible changes.
